generate/g_appcode.go

```diff
@@ -255,6 +256,9 @@ func (tag *OrmTag) String() string {
 	if len(ormOptions) == 0 {
 		return ""
 	}
+	if tag.Comment != "" {
+		return fmt.Sprintf("`orm:\"%s\" description:\"%s\"`", strings.Join(ormOptions, ";"), tag.Comment)
```

Comment:
description is for setting the swagger.Schema.Description.
https://github.com/beego/bee/blob/master/generate/swaggergen/g_docs.go#L988